packages/xo-web/src/common/xo/migrate-vm-modal/index.js

```diff
@@ -227,13 +227,17 @@ export default class MigrateVmModalBody extends BaseComponent {
       migrationNetworkId,
       targetSrs,
     } = this.state
+    const { $pool: poolId } = vm
     return (
       <div>
         <div className={styles.block}>
           <SingleLineRow>
             <Col size={4}>{_('migrateVmSelectHost')}</Col>
             <Col size={8}>
               <SelectHost
+                compareContainers={(pool1, pool2) =>
+                  pool1.id === poolId ? -1 : pool2.id === poolId ? 1 : 0
+                }
```

Comment:
This would also be useful [there](https://github.com/Rajaa-BARHTAOUI/xo-web/blob/vm-migrate-selector/packages/xo-web/src/xo-app/home/vm-item.js#L176).